addons/stock/static/src/views/search/stock_orderpoint_search_panel.js

```diff
@@ -9,11 +9,13 @@ export class StockOrderpointSearchPanel extends SearchPanel {
 
     setup() {
         super.setup(...arguments);
-        this.globalVisibilityDays = useState({value: 0});
+        const storedVal = localStorage.getItem("stock.orderpoint_horizon")
```

Comment:
Use sessionStorage instead. This will limit the time where this value is set for the user.